.travis.yml

```diff
@@ -39,26 +38,26 @@ jobs:
         - pip install --user --upgrade pip
         - pip install --user yamllint
         # install golangci-lint
+        # workaround for https://github.com/golangci/golangci-lint/issues/575
         - curl -sf
-          "https://install.goreleaser.com/github.com/golangci/golangci-lint.sh"
+          "https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh"
```

Comment:
I always cringe when using `master` branch versions of dependencies... letting this one pass through, and hopefully in the future we would stick to a version.

Reply:
will go back to the older version of code once the above-mentioned issue is fixed